core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/dataformats/DataFormatModel.java

```diff
@@ -72,7 +72,11 @@ public void setSupportedDataFormats(ArrayList<DataFormat> supportedDataFormats)
 	 * @return
 	 */
 	public DataFormat getDataFormatByName(String dataformat) {
-		return supportedDataFormats.stream().filter(df -> df.getName().equals(dataformat)).findFirst().orElse(null);
+		DataFormat foundDF = supportedDataFormats.stream().filter(df -> df.getName().equals(dataformat)).findFirst().orElse(null);
+		if (foundDF == null) {
+			foundDF = supportedDataFormats.stream().filter(df -> df.getModelName().equals(dataformat)).findFirst().orElse(null);
```

Comment:
I think getDataFormatByName is still accurate, but now it has a two-pronged approach. If name fails (which it seems to for data formats sometimes), we simply fall back to the model name. It only used getName() previously and this is limited to data formats, so my limited testing (and existing unit tests) have not raised any red flags yet.

I think getDataFormatByNameOrModelName, though more explicit, is an unnecessary step since the code is so simple at this point.